### PR TITLE
fix(ZNTA-2087) button-off styling for validations in frontend

### PR DESCRIPTION
**Pull request by kathryngo**
JIRA issue URL: https://zanata.atlassian.net/browse/ZNTA-2087

Styling for Off Button in Validation Settings of Zanata frontend

validation

Accidentally created on local branch with wrong Jira issue number, correct issue is ZNTA-2087

## Reviewer tasks

The following is based on the
[Zanata Development Guidelines](https://github.com/zanata/zanata-platform/wiki/Development-Guidelines).
As a reviewer, you should check the guidelines regularly for updates, and just
to refresh your memory.

The reviewer can use this list for reference to make sure
they have considered all the important points.

Use the checkboxes or not, whatever works best for you.

 - Code quality
   - [ ] Code passes style check (checkstyle/eslint)
   - [ ] Code is clear and concise
   - [ ] UI code is internationalised
   - [ ] Code has adequate comments where appropriate
   - Code is in an appropriate place
     - [ ] Classes are in appropriate packages
     - [ ] Code fits with class's responsibilities (SRP)
   - [ ] Configuration files are documented enough
   - If code is removed
     - [ ] No remaining code references the removed code
       - [ ] No orphaned rewrite rules
       - [ ] No orphaned config settings
     - [ ] No remaining comments refer to the removed code
     - [ ] Unused imports and libraries are removed
 - Testing
   - [ ] New code has tests
   - [ ] Changed code has tests
   - [ ] All tests pass
   - [ ] Test coverage stable or increased
 - Documentation
   - [ ] New features are documented
   - [ ] Docs removed for deleted features
   - [ ] Docs updated for all changed features
   - [ ] Developer/sysadmin docs updated if appropriate
 - If there are new dependencies
   - [ ] Does each new dependency pass all the
         [Considerations for new dependencies](https://github.com/zanata/zanata-platform/wiki/Development-Guidelines#new-technologies-and-dependencies)?

----
*This template can be updated in .github/PULL_REQUEST_TEMPLATE.md*

---
This change is [Reviewable](https://reviewable.io/reviews/zanata/zanata-platform/434)

### CODE REVIEW

File: server/zanata-frontend/src/frontend/app/styles/style.less [hunk 1]
```diff
@@ -6649,6 +6649,12 @@ table.tmx-table .badge .n1 {
   -moz-hyphens: initial;
   hyphens: initial;
 }
+
+.new-zanata .button--group input[type=radio]:checked+.button--Off {
+  background-color: @color-dark !important;
+  color: #f2f2f2;
```
Comment (aeng) [inline server/zanata-frontend/src/frontend/app/styles/style.less]: Is this color used for these copyTrans button only? or its a value from our color scheme?

Comment (kathryngo) [inline server/zanata-frontend/src/frontend/app/styles/style.less]: From the colour scheme

File: server/zanata-frontend/src/frontend/app/styles/style.less [hunk 2]
```diff
@@ -6650,6 +6650,11 @@ table.tmx-table .badge .n1 {
   hyphens: initial;
 }
 
+.new-zanata .button--group input[type=radio]:checked+.button--Off {
+  background-color: @color-dark !important;
+  color: #f2f2f2;
```
Comment (aeng) [inline server/zanata-frontend/src/frontend/app/styles/style.less]: What about the #f2f2f2? should we use variable for it too?

Comment (aeng) [inline server/zanata-frontend/src/frontend/app/styles/style.less]: Also make sure to replace any #f2f2f2 in frontend to use to variable too.

Comment (kathryngo) [inline server/zanata-frontend/src/frontend/app/styles/style.less]: Shall see if there is one similar, and if not I will create one.

Comment (kathryngo) [inline server/zanata-frontend/src/frontend/app/styles/style.less]: Yup, will do. Thanks.

**Comment by aeng** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/zanata/zanata-platform/434#-:-Kpr2GXmMC-02p3P6air:bnfp4nl)*

**Pull request merged**